### PR TITLE
Add Users API examples

### PR DESCRIPTION
**Pull request by S-H-GAMELINKS**
Hi.
Thanks for maintenance this great library.

I added Users API ezample in `__example/users`.
I'd be happy to merge them if you'd like these example.

### CODE REVIEW

File: .codeclimate.yml [hunk 1]
```diff
@@ -22,3 +22,4 @@ exclude_patterns:
   - "vendor/"
   - "_example/webinar.go"
   - "_example/add_group_member.go"
+  - "_example/users/*.go"
```
Comment (rafecolton) [inline .codeclimate.yml:25]: Please add a newline to the end of the file (not a blankline, just a newline) - it should be a setting in your editor

File: _example/users/delete_user.go [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+	zoom.APISecret = apiSecret
+
+	// Delete User by UserID or Email
+	opt := zoom.DeleteUserOptions{
+		EmailOrID: userID,
+	}
+
+	response := zoom.DeleteUser(opt)
+
+	log.Printf("Resopnse: %+v\n", response)
```
Comment (rafecolton) [inline _example/users/delete_user.go:27]: Calling the return value `response` is a bit misleading IMO as it is not a response object (i.e. a user), the return value is only error. 

Would like to see it more like this:

```golang
if err := zoom.DeleteUser(opt); err != nil {
    log.Printf("Error: %+v\n", err)
}

File: _example/users/get_user.go [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+	zoom.APIKey = apiKey
+	zoom.APISecret = apiSecret
+
+	// Get User info by UserID
+	opt := zoom.GetUserOpts{
+		EmailOrID: userID,
+	}
+
+	response, err := zoom.GetUser(opt)
```
Comment (rafecolton) [inline _example/users/get_user.go:26]: Please use `user` here instead of `response`

File: _example/users/get_user.go [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+		log.Printf("Error: %+v\n", err)
+	}
+	log.Printf("Resopnse: %+v\n", response)
+
+	// Get User info by Email
+	opt = zoom.GetUserOpts{
+		EmailOrID: userEmail,
+	}
+
+	response, err = zoom.GetUser(opt)
```
Comment (rafecolton) [inline _example/users/get_user.go:38]: Same, please use `user` here

File: _example/users/get_users_list.go [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+
+	zoom.APIKey = apiKey
+	zoom.APISecret = apiSecret
+
+	opt := zoom.ListUsersOptions{
+		PageSize:   15,
+		PageNumber: 1,
+	}
+
+	res, err := zoom.ListUsers(opt)
```
Comment (rafecolton) [inline _example/users/get_users_list.go:24]: Use a more descriptive variable name please (`listUsersResponse` perhaps or at least fully spell out `response`)

**Review by rafecolton — CHANGES_REQUESTED** [review]
Thanks for the contribution! Requesting a few minor changes 🙂

**Comment by S-H-GAMELINKS** [comment]
@rafecolton 
Thanks for your review comments!
It's done!

**Review by rafecolton — APPROVED** [review]

**Pull request merged**